### PR TITLE
Added redux-persist

### PR DESCRIPTION
**Pull request by UtopiaEH**
## Changelog
```changelog
 - NASA: Added redux-persist for localStorage
```
  
## Change Type
- [x] `Added`: new feature - non-breaking change which adds functionality.
- [ ] `Bug fix`: non-breaking change which fixes an issue.
- [ ] `Changed`: for changes in existing functionality.
- [ ] `Deprecated`: for soon-to-be removed features.
- [ ] `Documentation` update.
- [ ] `Performance`: for performance and optimization functionality.
- [ ] `Removed`: for now removed features.
- [ ] `Security`: in case of vulnerabilities.
## Required Checks
 - [ ] Documentation
 - [ ] Unit tests
 - [x] Tested on Chrome
 - [ ] Tested on Firefox
 - [ ] Tested on IE11
 - [x] npm run eslint
 - [x] npm run prettier
## Screenshots
________________

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
 import { Provider } from 'react-redux'
+import { PersistGate } from 'redux-persist/integration/react'
+
+import { store, persistor } from 'redux/redux-store'
+import { Preloader } from 'ui-kit'
 
-import store from './redux/redux-store'
 import App from './App'
 import * as serviceWorker from './serviceWorker'
 import './index.css'
```
Comment (UncleFairy) [inline src/index.js]: add empty line above `import './index.css'`

File: src/redux/redux-store.js [hunk 2]
```diff
@@ -2,21 +2,44 @@ import { createStore, applyMiddleware } from 'redux'
...
+      'yearEnd',
+    ]),
+    createBlacklistFilter('imgDailyPage', ['image']),
+  ],
+}
+
+const persistedReducer = persistReducer(persistConfig, rootReducers)
+
 const store = createStore(
-  rootReducers,
```
Comment (andreiturcanu) [inline src/redux/redux-store.js]: should you not pass only the persisted reducer to the createStore method?

Comment (UtopiaEH) [inline src/redux/redux-store.js]: this line is empty

Comment (UtopiaEH) [reply src/redux/redux-store.js]: this line is empty

**Pull request merged**